timetable/utils.py

```diff
@@ -48,8 +48,11 @@ def get_timetable_context(lessons):
 
     try:
         alternative_day_plan = DayPlan.objects.get(date=date.today())
-        alternative_periods = Period.objects.filter(schedule=alternative_day_plan.schedule)
-    except ObjectDoesNotExist:
+        if alternative_day_plan.schedule.is_default:
+            alternative_day_plan = None
+        else:
+            alternative_periods = Period.objects.filter(schedule=alternative_day_plan.schedule)
```

Comment:
What if `alternative_day_plan` says that lessons are cancelled? Note that `DayPlan.schedule` can be `None` [here](https://github.com/zorzalo3/zorza/blob/master/timetable/models.py#L70). Then you get `AttributeError`, because `NoneType` object has no attribute `is_default`. You should check whether `alternative_day_plan.schedule` is not `None`.
**And again: please test your changes before commiting!**

![image](https://github.com/zorzalo3/zorza/assets/44808028/dbc49539-2137-4a2e-aa46-6422d67763a6)

Reply:
Thanks @Igifigi, I did not realize that the option "---------" is a valid option. Fixed 
![image](https://github.com/zorzalo3/zorza/assets/26010258/fadb413c-ecf7-4679-ba7f-15c14a717744)